src/transformState.js

```diff
@@ -61,8 +61,34 @@
  * @param {Object} state
  * @param {Object[]} transforms
  */
+
 function transformState(state, transforms) {
-  // write code here
+  const linkToStateObj = state;
```

Comment:
```suggestion
```
No need to create `linkToStateObj`. Both `linkToStateObj` and `state` are links to the same object. You can change directly `state` fields